apps/website/src/pages/docs/anchor.mdx

```diff
@@ -9,19 +9,35 @@ thumbnail: Anchor
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
+import RelatedComponents from '~/components/RelatedComponents.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='typography-anchor' />
-
 <LiveExample src='Anchor.main.tsx' truncate={false}>
   <AllExamples.AnchorMainExample client:load />
 </LiveExample>
 
+## Variants
+
+### `as` prop
```

Comment:
This can go under "Usage" rather than Variants

also we can mention usage with the `Link` component from various routers (e.g. Nextjs router, React router)

Reply:
This has been moved to the `Usage` section, and mentioned of using the `Link` component for the Next.js and React routers have been mentioned